sdk/cognitiveservices/pom.xml

```diff
@@ -14,6 +14,13 @@
   <description>This package contains the parent module of Microsoft Azure Cognitive Services SDK.</description>
   <url>https://github.com/Azure/azure-sdk-for-java</url>
 
+  <!--<parent>
```

Comment:
Did you intentionally leave this here commented out?

Reply:
I'm going to remove it - @JonathanGiles  suggested that until we know about potential side effects of linking it to the parent POM that we should leave it as is.

Reply:
Removed.